public/index.html

```diff
@@ -31,15 +31,17 @@
           </ul>
         </ul>
       </div>
+      <!--
```

Comment:
The only reason I left it is because once the FxA storage server is up then we'll probably add this back in.